dat/missions/neutral/adblocker.lua

```diff
@@ -0,0 +1,164 @@
+--[[
+<?xml version='1.0' encoding='utf8'?>
+<mission name="Adblocker">
+ <unique />
+ <chance>20</chance>
+ <location>Bar</location>
```

Comment:
You should add the following line in the header:
```xml
<cond>return require("misn_test").reweight_active()</cond>
```
It just makes it so the probability of the mission appearing goes down the more active missions the player has.